debug_scripts/estimate_epoch_start_time.py

```diff
@@ -103,6 +120,10 @@ def predict_future_epochs(starting_epoch_timestamp, avg_epoch_length,
 
 # Main function to run the process
 def main(args):
+    if not is_valid_timezone(args.timezone):
```

Comment:
You can call validate straight in `parser.add_argument(...)` using the `type` argument. [docs](https://docs.python.org/3/library/argparse.html#type). You need to change the return value of the function.